src/pages/products/Filters.jsx

```diff
@@ -0,0 +1,155 @@
+import { useProductFilter } from '../../context/product-context'
+import productStyle from './Products.module.css'
+
+function Filters() {
+  const { filters, productDispatch } = useProductFilter()
+  const { category, rangeLimit, sortBy, rating } = filters
+  const { action, adventure, sports, strategy, arcade } = category
+  return (
+    <aside className={`${productStyle['filter']} text__md`}>
+      <div className={productStyle['filter__header']}>
+        <span className="font__bold">Filters</span>
+        <button
+          className={productStyle['filter__clear']}
+          onClick={() => productDispatch({ type: 'ClearAllFilters' })}>
+          Clear
+        </button>
+      </div>
+      <div
+        className={`filter__slider ${productStyle['flex__column']} text__md`}>
+        <label for="slider">Select your Price</label>
+        <div
+          className={`${productStyle['slider__price']} text__md font__light`}>
+          <span>1000</span>
+          <span>5500</span>
+          <span>10000</span>
+        </div>
+        <input
+          id="slider"
+          type="range"
+          min="1000"
+          max="10000"
+          value={rangeLimit}
+          className="slider"
+          onChange={e =>
+            productDispatch({
+              type: 'PRICE_RANGE',
+              payload: e.target.value,
+            })
+          }
+        />
+        <div className="slider__output text__md">
+          Selected Price is{' '}
+          <span className={productStyle['text__primary']}>₹ {rangeLimit}</span>
+        </div>
+      </div>
+      <div className={`filter__category ${productStyle['flex__column']}`}>
+        <h5>Category</h5>
+
+        <label className={productStyle['filter__names']}>
+          <input
+            type="checkbox"
+            checked={action}
+            onChange={() => productDispatch({ type: 'Action' })}
+          />
+          Action
+        </label>
+        <label className={productStyle['filter__names']}>
+          <input
+            type="checkbox"
+            checked={arcade}
+            onChange={() => productDispatch({ type: 'Arcade' })}
+          />
+          Arcade
+        </label>
+        <label className={productStyle['filter__names']}>
+          <input
+            type="checkbox"
+            checked={strategy}
+            onChange={() => productDispatch({ type: 'Strategy' })}
+          />
+          Strategy
+        </label>
+        <label className={productStyle['filter__names']}>
+          <input
+            type="checkbox"
+            checked={adventure}
+            onChange={() => productDispatch({ type: 'Adventure' })}
+          />
+          Adventure
+        </label>
+        <label className={productStyle['filter__names']}>
+          <input
```

Comment:
You can create reusable components for checkbox, radio, etc.

Reply:
sure will do it